templates/snap-details.html

```diff
@@ -101,6 +115,47 @@ <h4>Where people are using {{ snap_title }}</h4>
       whitelistUrls: ['snapcraft.io']
     }).install();
     Raven.context(function () {
+      if (typeof ga !== 'undefined') {
+        var ctrlDown = false;
+
+        function keyDown(e) {
+          var key = e.key.toLowerCase();
+          if (key === 'control' || key === 'meta') {
+            ctrlDown = true;
+          }
+        }
+
+        function keyUp(e) {
+          var key = e.key.toLowerCase();
```

Comment:
This doesn't work on Mac.

When 'Cmd' button (meta) is held clicking other buttons (like C) doesn't trigger keyup. This makes it harder to detect.
Keydown is triggered normally when Cmd is held, so possibly, if we move `if ((key === 'c' || key === 'x') && ctrlDown)` to keydown handler it should work...